tests/test_pytesseract.py

```diff
@@ -1,14 +1,21 @@
 # encoding: utf-8
 
 import os.path
-from sys import platform, version_info
 from multiprocessing import Pool
+from sys import platform, version_info
 
 import pytest
-
-from pytesseract.pytesseract import get_tesseract_version, image_to_data, \
-    image_to_string, image_to_boxes, image_to_pdf_or_hocr, image_to_osd, prepare
-from pytesseract.pytesseract import Output, TSVNotSupported
+from pytesseract import (
+    Output,
+    TSVNotSupported,
+    _prepare,
```

Comment:
Sorry for the nit-picking, but one additional thing from me: since _prepare is "private" function, we don't need to export it with the others.  I am commenting on this, just because it will be easier to maintain of we don't export the private functions and it will be more consistent.

Reply:
I'm fine with it 😄.